tests/sdk/sql2/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "1.5.0"
 edition = "2021"
 
 [dependencies]
-surrealdb = { path = "../../../lib", version = "1.5.0", features = ["kv-surrealkv", "sql2"] }
+surrealdb = { path = "../../../lib", version = "1.5.0", features = ["kv-tikv"] }
```

Comment:
Hey @emmanuel-keller 

In 1.5, the surrealKV version is not updated because of which certain bugs which were seen before might still persist. Thought of informing in case you are testing anything out

Reply:
Just needs to be updated to this version as in on main

https://github.com/surrealdb/surrealdb/blob/5e33d4c311220cc68aaf5e34153136bb4b337c2b/core/Cargo.toml#L146

If required, will open a PR on 1.5

Reply:
Indeed, as it is a different concern, it is better to make a separate PR